molecule_common/base/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos/systemd:latest
+
+RUN yum makecache fast && yum update -y && \
+    yum install -y python sudo yum-plugin-ovl && \
+    sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf
```

Comment:
Outside of the base image, the rest of the file is verbatim from Molecule. I will work on addressing the comments here and also upstream.